### ISSUE TITLE
Phase 4 tracking issue

### ISSUE THREAD

**Issue by dtig** [WebAssembly/relaxed-simd#134]
As the last vote for Phase 4 was provisional, and the outstanding PRs have been merged, is there anything else that's needed before this can be considered a full vote for Phase 4?

**Comment by rossberg** [comment]
Yeah, I'm afraid there still is a bit of work left to do for the spec, since the additions to the numerics section are still missing much of their prose semantics.

**Comment by dtig** [comment]
Thanks @rossberg, I'll leave this issue open till we can codify the provisional vote into a full Phase 4 vote.

**Comment by ngzhian** [comment]
#135 is merged.
I think all the spec work is done (modulo intended bugs).
FYI that there are 2 "merged" streams of work in this repo:
- `either` from threads proposal in the spec interpreter
- profiles from the profiles proposal.

**Comment by alexcrichton** [comment]
I'm a bit belated taking a look at this again, but I think it might be good to handle https://github.com/WebAssembly/relaxed-simd/issues/125 before phase 4. The proposed lowering on x64 of `pblendvb` does not match the semantics as specified in the spec currently.
